packages/venia-concept/src/components/Gallery/items.js

```diff
@@ -16,7 +16,9 @@ class GalleryItems extends Component {
             shape({
                 id: number.isRequired,
                 name: string.isRequired,
-                small_image: string.isRequired,
+                small_image: shape({
+                    url: string.isRequired
+                }),
```

Comment:
The `name`, `price`, and `small_image` prop types are unnecessary in this component. `GalleryItems` needs an array of objects that each have an `id` property in order to render successfully, but no more than that.

Reply:
Removed unnecessary props